webpack.config.js

```diff
@@ -12,7 +12,7 @@ const StyleLintPlugin = require('stylelint-webpack-plugin');
 
 Encore
   .setOutputPath('./src/Resources/public')
-  .setPublicPath('/bundles/sonataadmin')
+  .setPublicPath('.')
```

Comment:
did you try using /?

Reply:
This PR aims to fix path for resources that are not located into `/bundles/sonataadmin` by replacing the absolute by a relative path.

I don't think, generating an absolute path to the root directory will fix the issue.

Note the current version 3.x of sonata is using relative paths: https://github.com/sonata-project/SonataAdminBundle/blob/9a5b16bb43a845283d8eea6241cabe4671bfa817/src/Resources/public/css/styles.css#L12